setup.py

```diff
@@ -27,7 +27,9 @@ def get_version():
     'appdirs>=1.4.3,<2.0',
     'carto>=1.11.3,<2.0',
     'jinja2>=2.10.1,<3.0',
+    'numpy<=1.22.4',
```

Comment:
Why is this required?

Reply:
> With Python 3.8, the following test fails if numpy>1.22.4. This only affects Python 3.8 because Python 3.7 was dropped with NumPy 1.22.0. So I've also added the numpy<=1.22.4 requirement. [Job](https://github.com/CartoDB/cartoframes/actions/runs/3759768645/jobs/6389729223)

So with NumPy version `>1.22.4` (only available for Python 3.8 or higher), we get the following error when tests are executed:

```
  E   NotImplementedError: A polygon does not itself provide the array interface. Its rings do.
```
It seems a breaking change included in that version :thinking:

Reply:
I have tested with numpy 1.24.2 and the tests pass